Known_Issues/All VMMS operations are blocked on Silver or higher durability.md

```diff
@@ -0,0 +1,100 @@
+# VMMS operations are blocked node gets stuck in deletion and creation after deleting a VMSS
+
+## Problem
+- All VMSS operations are blocked on Silver or higher durability Scaleset after trying to delete a node from a VMMS 
+- The node being deleted (seed node) is stuck in Disabling state
+
+## Symptoms
+- One of the seed nodes is stuck disabling with Safety Check - **EnsureSeedNodeQuorum**
+- All operation started after the delete operation will be blocked until the deletion completes, including adding new VMs
```

Comment:
Suggestion: expand ", including adding new VMs" to include more examples
scaling operations, cancellation tasks, auto os upgrades, and other operational jobs.